src/main/java/seedu/address/ui/AddWindow.java

```diff
@@ -169,12 +179,21 @@ public void focus() {
     }
 
     /**
-     * Adds a Status for the newly created Person object.
+     * Retrieves the last index in {@code ReadOnlyAddressBook}'s list of Person.
+     * @return last index
      */
-    private void addStatusForNewPerson() {
+    private int getAbLastIndex() {
         ReadOnlyAddressBook ab = logic.getAddressBook();
```

Comment:
Maybe mention it is one based last index in the method name, since the Index class mentions whether it is one based or zero based index.